packages/blitz-rpc/package.json

```diff
@@ -40,7 +39,9 @@
     "react-dom": "18.0.0",
     "typescript": "^4.5.3",
     "unbuild": "0.7.6",
-    "watch": "1.0.2"
+    "vite": "3.0.4",
```

Comment:
Is there a reason for this?

Reply:
Sorry! That's was my mistake when I ran `pnpm test` in blitz-rpc npm/cli asked me to install vite that I didn't notice that it will add to package.json instead of global node_modules.